website on localhost is no longer working

### Story

_No response_

### Description

I can no longer run our website on localhost.  I really would like to understand why this is so unstable?




PS C:\Users\iansk\documents\github\website> npm install

added 1 package, removed 3 packages, changed 2 packages, and audited 2703 packages in 14s

304 packages are looking for funding
  run `npm fund` for details

17 vulnerabilities (4 moderate, 12 high, 1 critical)

To address issues that do not require attention, run:
  npm audit fix

To address all issues possible (including breaking changes), run:
  npm audit fix --force

Some issues need review, and may require choosing
a different dependency.

Run `npm audit` for details.
PS C:\Users\iansk\documents\github\website> npm start

> flowforge-website@1.0.0 start
> npm-run-all clean:dev build:js --parallel dev:*


> flowforge-website@1.0.0 clean:dev
> cross-env npx del-cli '_site/!(img)' && cross-env npx mkdirp '_site/js'


> flowforge-website@1.0.0 build:js
> terser -c -m -o _site/js/cc.min.js node_modules/vanilla-cookieconsent/dist/cookieconsent.js src/js/cookies.js


> flowforge-website@1.0.0 dev:postcss
> cross-env TAILWIND_MODE=watch npx postcss ./src/css/style.css -o ./_site/css/style.css --config ./postcss.config.js -w


> flowforge-website@1.0.0 dev:netlify
> npx netlify dev


> flowforge-website@1.0.0 dev:docs
> node scripts/dirExists ../flowforge/docs && nodemon -w ../flowforge/docs -e md --exec "npm run docs" || echo "docs not found - skipping..."

"docs not found - skipping..."

warn - You have enabled the JIT engine which is currently in preview.
warn - Preview features are not covered by semver, may introduce breaking changes, and can change at any time.

info - Tailwind CSS is watching for changes...
info - https://tailwindcss.com/docs/just-in-time-mode#watch-mode-and-one-off-builds
◈ Netlify Dev ◈
◈ Using simple static server because '[dev.framework]' was set to '#static'
◈ Running static server from "website\_site"
◈ Setting up local development server
​
> Installing plugins
   - @algolia/netlify-plugin-crawler@1.0.0
​
  Dependencies installation error
────────────────────────────────────────────────────────────────
​
  Error message
  Error while installing dependencies in C:\Users\iansk\Documents\GitHub\website/.netlify/plugins/
  npm ERR! code 1
  npm ERR! path C:\Users\iansk\Documents\GitHub\website\.netlify\plugins\node_modules\@algolia\netlify-plugin-crawler
  npm ERR! command failed
  npm ERR! command C:\WINDOWS\system32\cmd.exe /d /s /c [ -d dist/ ] || npm run build
  npm ERR! > @algolia/netlify-plugin-crawler@1.0.0 build
  npm ERR! > npx tsc -b
  npm ERR!
  npm ERR!
  npm ERR!
  npm ERR!                 This is not the tsc command you are looking for
  npm ERR!
  npm ERR!
  npm ERR! To get access to the TypeScript compiler, tsc, from the command line either:
  npm ERR!
  npm ERR! - Use npm install typescript to first add TypeScript to your project before using npx
  npm ERR! - Use yarn to avoid accidentally running code from un-installed packages
  npm ERR! '[' is not recognized as an internal or external command,
  npm ERR! operable program or batch file.
​
  npm ERR!     C:\Users\iansk\AppData\Local\npm-cache\_logs\2023-04-19T19_41_46_403Z-debug-0.log
​
  Resolved config
  build:
    command: npm run build
    commandOrigin: config
    edge_functions: C:\Users\iansk\Documents\GitHub\website\netlify\edge-functions
    publish: C:\Users\iansk\Documents\GitHub\website\_site
    publishOrigin: config
  plugins:
    - inputs: {}
      origin: config
      package: '@algolia/netlify-plugin-crawler'
 »   Error: Could not start local development server

Error while installing dependencies in C:\Users\iansk\Documents\GitHub\website/.netlify/plugins/
npm ERR! code 1
npm ERR! path C:\Users\iansk\Documents\GitHub\website\.netlify\plugins\node_modules\@algolia\netlify-plugin-crawler
npm ERR! command failed
npm ERR! command C:\WINDOWS\system32\cmd.exe /d /s /c [ -d dist/ ] || npm run build
npm ERR! > @algolia/netlify-plugin-crawler@1.0.0 build
npm ERR! > npx tsc -b
npm ERR!
npm ERR!
npm ERR!
npm ERR!                 This is not the tsc command you are looking for
npm ERR!
npm ERR!
npm ERR! To get access to the TypeScript compiler, tsc, from the command line either:
npm ERR!
npm ERR! - Use npm install typescript to first add TypeScript to your project before using npx
npm ERR! - Use yarn to avoid accidentally running code from un-installed packages
npm ERR! '[' is not recognized as an internal or external command,
npm ERR! operable program or batch file.

npm ERR!     C:\Users\iansk\AppData\Local\npm-cache\_logs\2023-04-19T19_41_46_403Z-debug-0.log

Error: Error while installing dependencies in C:\Users\iansk\Documents\GitHub\website/.netlify/plugins/
npm ERR! code 1
npm ERR! path C:\Users\iansk\Documents\GitHub\website\.netlify\plugins\node_modules\@algolia\netlify-plugin-crawler
npm ERR! command failed
npm ERR! command C:\WINDOWS\system32\cmd.exe /d /s /c [ -d dist/ ] || npm run build
npm ERR! > @algolia/netlify-plugin-crawler@1.0.0 build
npm ERR! > npx tsc -b
npm ERR!
npm ERR!
npm ERR!
npm ERR!                 This is not the tsc command you are looking for
npm ERR!
npm ERR!
npm ERR! To get access to the TypeScript compiler, tsc, from the command line either:
npm ERR!
npm ERR! - Use npm install typescript to first add TypeScript to your project before using npx
npm ERR! - Use yarn to avoid accidentally running code from un-installed packages
npm ERR! '[' is not recognized as an internal or external command,
npm ERR! operable program or batch file.

npm ERR!     C:\Users\iansk\AppData\Local\npm-cache\_logs\2023-04-19T19_41_46_403Z-debug-0.log
    at runCommand (file:///C:/Users/iansk/Documents/GitHub/website/node_modules/netlify-cli/node_modules/@netlify/build/lib/install/main.js:20:24)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async installMissingPlugins (file:///C:/Users/iansk/Documents/GitHub/website/node_modules/netlify-cli/node_modules/@netlify/build/lib/install/missing.js:16:5)
    at async handleMissingPlugins (file:///C:/Users/iansk/Documents/GitHub/website/node_modules/netlify-cli/node_modules/@netlify/build/lib/plugins/resolve.js:97:5)
    at async resolvePluginsPath (file:///C:/Users/iansk/Documents/GitHub/website/node_modules/netlify-cli/node_modules/@netlify/build/lib/plugins/resolve.js:32:29)
    at async tGetPluginsOptions (file:///C:/Users/iansk/Documents/GitHub/website/node_modules/netlify-cli/node_modules/@netlify/build/lib/plugins/options.js:12:29)
    at async tGetPluginsOptions (file:///C:/Users/iansk/Documents/GitHub/website/node_modules/netlify-cli/node_modules/@netlify/build/lib/time/main.js:18:63)
    at async initAndRunBuild (file:///C:/Users/iansk/Documents/GitHub/website/node_modules/netlify-cli/node_modules/@netlify/build/lib/core/build.js:243:66)
    at async runAndReportBuild (file:///C:/Users/iansk/Documents/GitHub/website/node_modules/netlify-cli/node_modules/@netlify/build/lib/core/build.js:145:165)
    at async tExecBuild (file:///C:/Users/iansk/Documents/GitHub/website/node_modules/netlify-cli/node_modules/@netlify/build/lib/core/build.js:92:136)
ERROR: "dev:netlify" exited with 1.
PS C:\Users\iansk\documents\github\website>

--- a/package.json
+++ b/package.json
@@ -26,6 +26,7 @@
         "@11ty/eleventy-img": "^3.0.0",
         "@11ty/eleventy-plugin-rss": "^1.2.0",
         "@11ty/eleventy-plugin-syntaxhighlight": "^4.2.0",
+        "@algolia/netlify-plugin-crawler": "^1.0.15",
         "@flowforge/forge-ui-components": "^0.2.2",
         "@kevingimbel/eleventy-plugin-mermaid": "^2.0.0",
         "@tailwindcss/typography": "^0.4.1",

